modules/openexr/3.2.0/patches/module_dot_bazel.patch

```diff
@@ -0,0 +1,14 @@
+--- MODULE.bazel
++++ MODULE.bazel
+@@ -1,9 +1,10 @@
+ module(
+     name = "openexr",
+-    version = "0.0.0", # The version "0.0.0" is set here and will be patched within the Bazel Central Registry
++    version = "3.2.0",
+     compatibility_level = 1,
+ )
+ 
+ bazel_dep(name = "bazel_skylib", version = "1.4.2")
+ bazel_dep(name = "imath", repo_name = "Imath", version = "3.1.9")
```

Comment:
this line probably needs to be patched (the original file didn't have `repo_name=...`)

Reply:
another iteration...